code/datums/action.dm

```diff
@@ -673,7 +673,7 @@
 //Stickmemes
 /datum/action/item_action/stickmen
 	name = "Summon Stick Minions"
-	desc = "Allows you to summon faithful stickmen allies to aide you in battle."
+	desc = "Allows you to summon faithful stickmin allies to aid you in battle."
```

Comment:
typo

Reply:
ahh ive gotten so used to saying stickmin from the henry stickmin series heh